content/english/events/2019-08-08-hip-erg.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "You Belong. Overcoming Imposter Syndrome"
+description: "Join Techqueria ATX and HIP Austin BRG for an afternoon of deep discussions around overcoming imposter syndrome."
+date: 2019-08-08
+link: https://www.eventbrite.com/e/you-belong-overcoming-imposter-syndrome-tickets-67228741977/
+image: "/assets/img/events/2019-08-08-atx-hip.jpeg"
+locations:
+  - austin
+organizations:
+  - "HIP Austin BRG"
+organizers:
+  - "Darlene Santoyo"
+  - "Andrea Cardona"
+---
+
+We will be hosting a panel where our guests will delve into feelings of not belonging or being in over your head and how they have overcome those situations. Come on out ! There will be light snacks and fun networking!
+
+Date: Thursday, August 8, 2019
```

Comment:
@scarlettperry can we fix formatting for this one so there's a break between lines - i think this was case for other PR so if we could update - that would be 💯 

also - darlene is not added as a team member yet but if you wanted to add her, feel free!

also - HIP Austin BRG is not the organization - IBM is and we would want it to be added as an organization - like https://techqueria.org/supporters/lyft